src/Request.php

```diff
@@ -49,7 +49,10 @@ private static function clone($value)
     private $tlsHandshakeTimeout = 10000;
 
     /** @var int */
-    private $transferTimeout = 10000;
+    private $transferTimeout = 60000;
+
+    /** @var int */
+    private $inactivityTimeout = 5000;
```

Comment:
I think I'd set this equal to the `transferTimeout` for now, so `10000`, keeping everything as is for existing users.

Reply:
For BC I suppose, though as I said above, different timeouts seems more practical IMO.

Reply:
Yes, we can definitely change those in v5.

Reply:
Sounds good.

Reply:
What timeouts would you recommend? A docblock comment like "The default value will be changed to xxx in v5" would be nice.

Reply:
@enumag It mostly depends on the situation you're using the client in. It might be a good idea to give some pre-configurations as part of `HttpClientBuilder`.